gtsfm/loader/one_d_sfm_loader.py

```diff
@@ -0,0 +1,162 @@
+"""Loader of dataset used in 1DSFM and Colmap paper
```

Comment:
nit: "Loader for datasets used in 1DSFM and Colmap papers."

Reply:
Done